tests/module/utils/test_deprecation.py

```diff
@@ -1,23 +1,13 @@
 from __future__ import annotations
 
-import logging
-
-import pytest
-
 from manim.utils.deprecation import deprecated, deprecated_params
 
 
-def _get_caplog_record_msg(warn_caplog_manim):
-    logger_name, level, message = warn_caplog_manim.record_tuples[0]
+def _get_caplog_record_msg(manim_caplog):
+    logger_name, level, message = manim_caplog.record_tuples[0]
```

Comment:
Thoughts on making this function return `(level, message)` (`tuple[int, str]`) so that we can make sure it's still a warning?

Reply:
In principle I don't have a strong opinion for or against additionally testing that the message has been logged with the correct loglevel -- but I do wonder whether it is worth the effort of adapting all the tests for this particular module.

Reply:
Makes sense, I'm fine with not testing the level as long as the message is shown to the user.